framework/permissions.py

```diff
@@ -97,10 +97,10 @@ def feature_edit_list(user: User) -> list[int]:
     return []
 
   # Query features to find which can be edited.
-  features_editable = feature_helpers.get_all(
-    filterby=('can_edit', user.email()))
+  editable_feature_keys = feature_helpers.get_all(
+      filterby=('can_edit', user.email()), keys_only=True)
```

Comment:
It looks like the `get_all()` function's return type was mistyped (presumably by me 😅). and it actually is returning a list of keys or a list of verbose FeatureEntry dictionaries, so the return type needs to change to: `list[dict | ndb.Key]` - I think that should fix the mypy problems.